packages/astroport-governance/src/assembly.rs

```diff
@@ -370,22 +377,26 @@ pub struct ProposalListResponse {
 }
 
 pub mod helpers {
+    use crate::assembly::LINK_REGEX;
     use cosmwasm_std::{StdError, StdResult};
+    use regex::Regex;
 
-    const UNSAFE_CHARS: [char; 6] = ['<', '>', ' ', '\\', '{', '}'];
+    const ALLOWED_LINK_SIGNS: &str = "./:?=&_-#";
 
     /// Checks if the link is valid. Returns a boolean value.
-    pub fn is_valid_link(link: &str) -> bool {
+    pub fn is_valid_link(link: &str, regex: &Regex) -> bool {
         link.chars()
-            .all(|c| c.is_ascii() && !UNSAFE_CHARS.contains(&c))
+            .all(|c| c.is_ascii_alphanumeric() || ALLOWED_LINK_SIGNS.contains(c))
```

Comment:
I think we dont need these checks anymore since regexp does this job.

Reply:
Ok, removed redundant checks, thanks